universal-login-react/test/core/services/WalletStorageService.ts

```diff
@@ -0,0 +1,70 @@
+import {StorageService, WalletStorageService} from '../../../src';
+import {ApplicationWallet, CounterfactualWallet} from '@universal-login/commons';
+import sinon from 'sinon';
+import {expect} from 'chai';
+
+describe('WalletStorageService', () => {
```

Comment:
Not sure if those tests test a lot. I would consider writing MemoryStorageService, so we can write round trips tests.

Reply:
👍